v0/util/index.js

```diff
@@ -1313,21 +1313,37 @@ class CustomError extends Error {
  * @param {*} destination
  * @param {*} transformStage
  */
-function generateErrorObject(error, destination, transformStage) {
+function generateErrorObject(error, destination = "", transformStage) {
   // check err is object
   // filter to check if it is coming from cdk
-  if (error.fromCdk) {
-    return error;
-  }
+  // if (error.fromCdk) {
+  //   if (error.statTags && error.statTags.destination) {
+  //     error.statTags.destType = error.statTags.destination.toUpperCase();
+  //     delete error.statTags.destination;
+  //   }
+  //   return error;
+  // }
   const { status, message, destinationResponse } = error;
   let { statTags } = error;
   if (!statTags) {
     statTags = {
-      destination,
+      destType: destination,
       stage: transformStage,
       scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.EXCEPTION.SCOPE
     };
   }
+  // In previous versions of cdk, we had destination tag
+  // This block is mainly to support it and also for backward for non-cdk destinations
+  if (statTags.destination) {
+    const destType = statTags.destination
+    statTags = Object.keys(statTags).filter(k => k !== 'destination')
+      .reduce((prevMap, k) => ({ ...prevMap, [k]: statTags[k] }), {})
+    statTags.destType = destType;
+  }
```

Comment:
if (statTags.destination) {
   statTags.destType = statTags.destination;
   delete statTags.destination;
}

Reply:
This is done. Can you check and confirm ?